### DISCUSSION TITLE
Install manuelle²

### DISCUSSION THREAD

**Discussion by domguard** [picsou83/compta.libremen.com#17] [Q&A]
Bonjour
depuis mon dernier bilan Oracle Free tier a fermé mon compte...j'ai une sauvegarde de la BDD...
Du coup je reinstalle tout sur une clé USB avec une debian Live. 
Est-ce que la mention "debian 11" pour l'install manuelle est toujours valide ou je peux prendre la 12 ?
Merci!

**Comment by picsou83** [comment]
Bonjour, je pense que vous parlez de la procédure d'installation : Linux installation Hub Docker => B. Avec une debian 11.

Dans la mesure où vous utilisez Docker à l'intérieur de Debian, il n'y a pas de problème car le fichier Dockerfile utilise ensuite l'image Debian 11. Je testerai à l'occasion un Dockerfile avec Debian 12.

**Comment by domguard** [comment]
Oui, bien sur...l'avantage de docker :)
par contre, j'ai une sauvegarde de début mars 2024, donc je pense en version 1.109
Il faudrait donc que j'installe celle-ci PUIS mise a jour en v1.110, c'est ça ?
Dans ce cas, dans la methode Hub Docker, j'imagine que je dois changer la commande
`docker pull picsou83/compta-libremen-com:latest`
et mettre autre chose que "latest" ?

**Reply by domguard** [reply]
je vais passer par l'install manuelle, ça sera plus simple

**Comment by picsou83** [comment]
Non, pas besoin. Toujours installer la dernière version de l'image docker disponible et restaurer votre dernière sauvegarde de base de données. La base de données se mettra automatiquement à jour quelle que soit la version.

**Reply by domguard** [reply]
ça a très bien fonctionné, merci !
il me reste a retrouver les ajustements de formule et à les entrer dans la nouvelle interface
